test/integration/users.test.js

```diff
@@ -354,4 +354,36 @@ describe("Users", function () {
         });
     });
   });
+  describe("POST /users/verify", function () {
+    it("Should queue the Request", function (done) {
+      chai
+        .request(app)
+        .post("/users/verify")
+        .set("cookie", `${cookieName}=${jwt}`)
+        .end((err, res) => {
+          if (err) {
+            return done();
+          }
+          expect(res).to.have.status(200);
+          expect(res.body).to.be.a("object");
+          expect(res.body.message).to.equal("Your request has been queued successfully");
+          return done();
+        });
+    });
```

Comment:
Can we also add assertions to check if user `identityStatus` is updated to `PENDING`?

Reply:
that will be automatically checked if there is no error